### ISSUE TITLE
Tool does not exist

### ISSUE THREAD

**Issue by danmyhre** [MicrosoftDocs/OfficeDocs-Exchange#1557]
This does not exist in EXO admin center or in Exchange 2016 admin center

1. In the EAC, go to Tools &gt; Checks and click Check your on-premises Exchange Server with the Office 365 Best Practices Analyzer.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 13b8d4e5-6aae-5174-d924-b1d550d534b5
* Version Independent ID: d9b4b71c-b5d0-2462-8c44-9b3d6a4d4dd0
* Content: [About the Office 365 Best Practices Analyzer for Exchange Server](https://docs.microsoft.com/en-us/exchange/about-the-office-365-best-practices-analyzer-for-exchange-server#feedback)
* Content Source: [Exchange/ExchangeHybrid/about-the-office-365-best-practices-analyzer-for-exchange-server.md](https://github.com/MicrosoftDocs/OfficeDocs-Exchange/blob/public/Exchange/ExchangeHybrid/about-the-office-365-best-practices-analyzer-for-exchange-server.md)
* Product: **exchange-server-it-pro**
* Technology: ****
* GitHub Login: @mattpennathe3rd
* Microsoft Alias: **v-mapenn**

**Comment by parlevjo2** [comment]
I even do not have an Tools Section in Exchange Server 2016 CU14
Name                : XXXXXXXXXXXX
Edition             : Coexistence
AdminDisplayVersion : Version 15.1 (Build 1847.3)

**Comment by mattpennathe3rd** [comment]
@danmyhre @parlevjo2 Can you please verify that you meet all the requirements in the Prerequisites section of this document?

**Comment by parlevjo2** [comment]
Hi,

Thanks for the response. All the requirements of the Prerequisites section are met.

**Comment by parlevjo2** [comment]
On a Exchange Server 2013 I see the Tools section, but there the link is not working.
http://go.microsoft.com/?LinkId=9839201 => http://bestpracticesanalyzer.microsoft.com/clickonce/setup.exe
But the domain bestpracticesanalyzer.microsoft.com does not even exists anymore

**Comment by scanum** [comment]
@danmyhre Thank you for submitting feedback. We understand that this issue has been resolved.
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.

**Comment by danmyhre** [comment]
This isn't resolved @scanum

**Comment by mattpennathe3rd** [comment]
@danmyhre @parlevjo2 My apologies for the premature closure.

Regarding your issue, this article is slated for removal in the next few days because this analyzer no longer exists. (The Tools link to this article from within Server 2013 will still reach the page, but the article has been mostly wiped and now only explains that the analyzer is no longer available. The link to the analyzer is dead, unfortunately.) This tool was only ever intended for Exchange 2013, and as far as I know, it never got fully out of beta testing. There is no replacement for it.

In short, you should not worry about this as it is long dead and gone. Let me know if this is causing you trouble in some way; I had an extensive discussion with some of the technical people within Microsoft and our understanding is that this should not be needed in anyone's environment.
